### ISSUE TITLE
Error crawling rumble.com

### ISSUE THREAD

**Issue by mgnoonan** [mgnoonan/RssFeeder#211]
Looks like maybe rumble does not always have the target element. Need to handle this more cleanly.

Example data:
```json
{
    "@t": "2021-10-30T15:06:26.6477545Z",
    "@mt": "Attempting HTML tag parsing using body selector '{bodySelector}' and paragraph selector '{paragraphSelector}'",
    "@m": "Attempting HTML tag parsing using body selector '#videoPlayer' and paragraph selector 'video'",
    "@i": "2a4d07d6",
    "bodySelector": "#videoPlayer",
    "paragraphSelector": "video",
    "siteName": "rumble",
    "urlHash": "039c7752ada8ad6de5c62c257ed9b0aa",
    "url": "https://t.co/Ou9HZQXgRu",
    "collectionName": "bad-blue"
}
```

![image](https://user-images.githubusercontent.com/3255096/139540404-f1bd8af0-ec4b-4df2-8574-c50c72bf26e6.png)

**Comment by mgnoonan** [comment]
It's possible this was resolved in #221, waiting a few days to see if any rumble issues are still happening.

**Comment by mgnoonan** [comment]
The solution was to configure rumble.com crawls to always use the JsonLdTagParser. Looks like the ld+json format is becoming more popular, but unfortunately, I am having trouble getting the c# parser library to work properly in net6.
